configfiles/fit/config_Dplus_Fit_pp5TeV.yml

```diff
@@ -1,5 +1,5 @@
 pp5TeVPrompt:
-    Meson: 'Dplus'
+    Particle: 'Dplus'
```

Comment:
This does not work unless you include also the modification to the https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/GetRawYieldsDplusDs.C and https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/GetRawYieldsDplusDs.py scripts

Reply:
I did it but I had some problems with git history, I didn't realize those files were not included. They will be updated in the next commit.